docs/benchmarks/fio.md

```diff
@@ -25,9 +25,9 @@ You can find [configuration examples](https://github.com/xridge/kubestone/tree/m
 
 
 ## Sample benchmark
-To run a sample benchmark with PVC mode, the following command can be used:
+To run a sample fio benchmark, the following command can be used:
 ```bash
-$ kustomize build github.com/xridge/kubestone/config/samples/fio/overlays/pvc | kubectl create --namespace kubestone -f -
+$ kustomize build github.com/xridge/kubestone/config/samples/fio/base | kubectl create --namespace kubestone -f -
```

Comment:
Based on kustomize's documentation you don't need to specify `base`. So this will also work:
`kustomize build github.com/xridge/kubestone/config/samples/fio/ | kubectl create --namespace kubestone -f -`

Reply:
Well, it's not true.
```
$ kustomize build config/samples/fio
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/gnovak/Dev/xridge/kubestone/config/samples/fio'

Reply:
sorry. Seems like the kustomize repo puts the base into the top directory (and not under base): https://github.com/kubernetes-sigs/kustomize